lib/ui/src/modules/ui/libs/filters.test.js

```diff
@@ -74,6 +74,20 @@ describe('manager.ui.libs.filters', () => {
       expect(res).toEqual([stories[1], stories[2], stories[0]]);
     });
 
+    test('should sort nested stories by kind', () => {
```

Comment:
The name of a test is a bit misleading, it should be something like "should sort nested stories", because terminologically it's not "by kind".

Reply:
✅

Reply:
Looks like you changed a wrong test 🤣